### PR TITLE
Settings UX

### PR DESCRIPTION
**Pull request by stulees**
- bower updated to pull latest dependencies and remove some unnecessary resolutions
- gulp file
	- revising source tasks to use layout HTML files
	- updated "images" task to include settings layout images
	- updated "rise-rss" task to revise dependencies
- config files updated to include use of new angular module `config` for settings to obtain url values for layout HTML files
- settings
	- `settings.html` updated with complete settings and adheres to new UX
	- layout images added to project
	- controller updated with necessary logic and default settings values
	- app updated to revise module dependencies
	- e2e tests added and revised
- widget
	- `widget.html` removed and 3 layout HTML files added in replacement
	- layout HTML temporarily make use of `rise-rss` web component for obtaining data, this will eventually be replaced by `rise-rss-display`
	- `main.js` updated to correctly listen for `WebComponentsReady` event now that Polymer `1.0` being used
	- `rss.js` and `component.js` updated to temporarily implement logic for working with `rise-rss`, this will eventually be changed to work with `rise-rss-display`
- dist/ files added

### CODE REVIEW

**Comment by stulees** [comment]
Review - http://rva-test.appspot.com/#/PRESENTATION_MANAGE/id=7475be3f-6894-4f33-a51d-8ecc0a2bc691?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443

@donnapep Apologies for a big PR, but it's really just a lot of dist/ files. Please review, thanks.

@mmeiers You'll want to just review the `dist/settings.html` file as that's all the UX pertains to. Something to note is I had to add an embedded style `.overlay` and use it on a `<div>` so that there is an overlay between RVA and the widget settings. I couldn't find a style in common-style that I could use for this, so for now it's embedded. If you can recommend one or if you want to take mine and incorporate it into your UX prototype and add a style to common-style to use, then in a further PR for this Widget I can update it to use that style from common-style. 

Thanks

**Comment by donnapep** [comment]
@stulees Can the layouts have more descriptive names so that it's more intuitive? Maybe use the same names as the images?

**Comment by stulees** [comment]
@donnapep Yeah I couldn't think of suitable names. I'll go with the same names as the images and I'll make the fix in this PR.

**Comment by mmeiers** [comment]
:+1: 
 Regarding .overlay:
Im currently working on spreadsheet settings, I'll address it with that and see if i can find a replacement or commit it to common-style. Then we can do another pr for this one.

**Comment by stulees** [comment]
@donnapep Layout names have changed throughout to reflect the same names as the images (4x1, 2x1, 1x2). Can you review again, thanks

**Comment by stulees** [comment]
@mmeiers Sounds good, thanks

**Comment by donnapep** [comment]
:+1:

File: src/settings.html [hunk 1]
```diff
@@ -15,35 +16,205 @@
...
-          </h2>
-          <div class="sticky-wrapper">
-            <widget-button-toolbar
-              save="saveSettings()" cancel="closeSettings()" disable-save="settingsForm.$invalid"
-              contribute="https://github.com/Rise-Vision/widget-rss"></widget-button-toolbar>
-          </div>
-        </div>
+<div class="overlay"></div>
+<div class="container-fluid">
+  <div class="modal-dialog modal-lg">
```
Comment (ghislaineguerin) [inline src/settings.html]: @stulees why are we building this using the modal classes? should that be the modal from angular bootstrap?

Comment (stulees) [inline src/settings.html]: @ghislaineguerin RVA doesn't launch the widget settings by way of an Angular Bootstrap modal. The widget settings.html page is loaded by RVA into an `<iframe>`.  So in our widget settings.html page we need to define markup to use modal structure/classes to contain the settings content and give the whole appearance of a modal.

**Pull request merged**